components/admin_console/manage_tokens_modal/manage_tokens_modal.jsx

```diff
@@ -51,11 +51,11 @@ export default class ManageTokensModal extends React.PureComponent {
         this.state = {error: null};
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
+    componentDidUpdate(prevProps) { // eslint-disable-line camelcase
         const userId = this.props.user ? this.props.user.id : null;
-        const nextUserId = nextProps.user ? nextProps.user.id : null;
-        if (nextUserId && nextUserId !== userId) {
-            this.props.actions.getUserAccessTokensForUser(nextUserId, 0, 200);
+        const prevUserId = prevProps.user ? prevProps.user.id : null;
+        if (userId && prevUserId !== userId) {
+            this.props.actions.getUserAccessTokensForUser(prevUserId, 0, 200);
```

Comment:
I think this call should use the new user, so the variable `userId` instead of the `prevUserId`